package.json

```diff
@@ -223,30 +164,89 @@
+    "babel-plugin-transform-react-remove-prop-types": "^0.4.2",
+    "babel-polyfill": "^6.13.0",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-react": "^6.11.1",
+    "babel-preset-stage-2": "^6.13.0",
+    "babel-register": "^6.18.0",
     "blob-polyfill": "^2.0.20171115",
     "camelcase-keys-recursive": "^0.8.2",
-    "choma": "^1.1.0",
-    "cli-table": "^0.3.1",
+    "cheerio": "^1.0.0-rc.2",
+    "classlist-polyfill": "^1.0.3",
+    "command-line-args": "^3.0.1",
+    "css-loader": "^0.28.10",
     "downshift": "^1.22.5",
+    "enhanced-resolve": "^0.7.6",
     "eslint-plugin-import": "^2.7.0",
     "eslint-stats": "^1.0.0",
+    "extract-text-webpack-plugin": "next",
     "fast-levenshtein": "^2.0.6",
     "file-loader": "^1.1.11",
+    "find": "^0.2.9",
+    "fine-uploader": "^5.11.8",
+    "get-env": "^0.4.0",
+    "ismobilejs": "^0.4.0",
+    "jsdom": "^11.1.0",
+    "leaflet": "^1.0.3",
     "js-yaml": "^3.12.0",
     "local-storage-fallback": "^4.1.1",
-    "nyc": "^11.1.0",
+    "mapbox": "1.0.0-beta7",
+    "metalsmith": "^2.1.0",
+    "metalsmith-assets": "^0.1.0",
+    "metalsmith-broken-link-checker": "^1.0.1",
+    "metalsmith-collections": "^0.7.0",
+    "metalsmith-date-in-filename": "^0.0.14",
+    "metalsmith-filenames": "^1.0.0",
+    "metalsmith-in-place": "^1.4.4",
+    "metalsmith-layouts": "^1.6.5",
+    "metalsmith-markdownit": "^0.4.0",
+    "metalsmith-move-remove": "0.0.1",
+    "metalsmith-navigation": "^0.2.9",
+    "metalsmith-permalinks": "^0.5.0",
+    "metalsmith-sitemap": "^1.0.0",
+    "metalsmith-watch": "^1.0.3",
+    "node-sass": "^4.5.3",
+    "null-loader": "^0.1.1",
     "prop-types": "^15.6.1",
-    "puppeteer": "^1.9.0",
+    "react": "^15.5.4",
+    "react-addons-css-transition-group": "^15.3.2",
+    "react-addons-perf": "^15.3.0",
+    "react-autosuggest-ie11-compatible": "^8.0.6",
     "react-cropper": "^1.0.1",
+    "react-datepicker": "^0.29.0",
+    "react-dom": "^15.5.4",
     "react-dropzone": "^4.2.5",
+    "react-leaflet": "^1.2.0",
+    "react-redux": "4.4.8",
+    "react-router": "3",
+    "react-scroll": "1.5",
+    "react-tabs": "^1.0.0",
+    "redux": "^3.5.2",
+    "redux-thunk": "^2.1.0",
     "reselect": "^2.5.4",
+    "sass-loader": "^6.0.6",
+    "style-loader": "^0.20.2",
+    "svg-url-loader": "^2.3.2",
     "sync-request": "^6.0.0",
     "text-loader": "^0.0.1",
+    "tinyliquid": "^0.2.33",
+    "url-loader": "^0.6.2",
     "url-search-params": "^0.10.0",
     "us-forms-system": "https://github.com/usds/us-forms-system.git#f476ed0ff038403d04e531bbbae41f72290e23b5",
+    "uswds": "^1.4.2",
     "vanilla-lazyload": "^8.17.0",
     "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#6490dc3ffe58717846dc45697d3fb59a2b5fba3c",
-    "webpack-bundle-analyzer": "^2.11.1"
+    "webpack": "^4.1.1",
+    "webpack-bundle-analyzer": "^2.11.1",
+    "webpack-dev-server": "^3.1.1",
+    "webpack-manifest-plugin": "^2.0.0-rc.2"
   }
 }
```

Comment:
This is awesome! But wondering, what's the general method of knowing whether a dependency is a `devDependency`? I was thinking before that Webpack and everything that sits behind it would be a `devDependency`, leaving mostly just Metalsmith and its plugins as top-level dependencies because those modules are required for every deployment.